packages/solutions/app-tools/src/plugins/deploy/utils.ts

```diff
@@ -0,0 +1,72 @@
+import path from 'path';
+import { ROUTE_SPEC_FILE, fs as fse, isDepExists } from '@modern-js/utils';
+import { ServerRoute } from '@modern-js/types';
+import { IAppContext } from '@modern-js/core';
+
+export type ServerAppContext = {
+  sharedDirectory: string;
+  apiDirectory: string;
+  lambdaDirectory: string;
+  metaName: string;
+};
+
+export const serverAppContenxtTemplate = (appContext: IAppContext) => {
+  const {
+    appDirectory,
+    sharedDirectory,
+    apiDirectory,
+    lambdaDirectory,
+    metaName,
+  } = appContext;
+  return {
+    sharedDirectory: `path.join(__dirname, "${path.relative(
+      appDirectory,
+      sharedDirectory,
+    )}")`,
+    apiDirectory: `path.join(__dirname, "${path.relative(
+      appDirectory,
```

Comment:
emmm, can we just use `.` or '/'

Reply:
I'm not sure we talk the same thing. I mean the code:
`${path.relative(appDirectory, appDirectory)`